docs/home-configuration-nix-header.5

```diff
@@ -0,0 +1,9 @@
+.TH "HOME-CONFIGURATION\&.NIX" "5" "01/01/1980" "Home Manager"
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.\" enable line breaks after slashes
+.cflags 4 /
+.SH "NAME"
+\fIhome\-configuration\&.nix\fP \- Home Manager configuration specification
```

Comment:
I think this should also include the `DESCRIPTION` and `OPTIONS` headers. Compare
![swappy-20231129_235403](https://github.com/nix-community/home-manager/assets/798147/638a09b7-0231-4b0b-8020-525e6f7a0805)
and
![swappy-20231129_235424](https://github.com/nix-community/home-manager/assets/798147/25054300-31b7-49b5-a6df-7d40c7f8c5fc)

The description section is not so important, it should probably be reworded anyway, but it would be nice to have the `OPTIONS` header.